packages/shared/status/statuses.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
-import { useTranslation } from 'react-i18next';
 import { InProgressIcon } from '@patternfly/react-icons';
+import { useSharedTranslation } from '../useTranslationHook';
```

Comment:
nits: useTranslation is already provided by i18n... `useCustomTranslationHook` or `useSharedTranslationHook` as a name for the file would be better IMO.
```suggestion
import { useSharedTranslation } from '../useSharedTranslationHook';
```

Reply:
ack